src/main/java/uk/ac/cam/cl/dtg/isaac/configuration/IsaacApplicationRegister.java

```diff
@@ -64,8 +64,7 @@ public IsaacApplicationRegister() {
         SegueGuiceConfigurationModule segueGuiceConfigurationModule = new SegueGuiceConfigurationModule();
         IsaacGuiceConfigurationModule isaacGuiceConfigurationModule = new IsaacGuiceConfigurationModule();
         
-        injector = Guice.createInjector(new SchoolLookupConfigurationModule(),
-                isaacGuiceConfigurationModule, segueGuiceConfigurationModule);
+        injector = Guice.createInjector(isaacGuiceConfigurationModule, segueGuiceConfigurationModule);
```

Comment:
Why does this remove the school one?

Reply:
I moved it into the isaac guice configuration module to tidy things up. This comment did remind me to delete the SchoolLookupConfigurationModule though, thanks!